packages/scene-composer/src/components/panels/SceneHierarchyPanel/SceneHierarchyDataProvider.tsx

```diff
@@ -192,7 +197,7 @@ const SceneHierarchyDataProvider: FC<SceneHierarchyDataProviderProps> = ({ selec
         // remove child ref from parent
         if (!oldParentRef) {
           const newRoots = document.rootNodeRefs.filter((ref) => ref !== objectRef);
-          updateDocumentInternal({ rootNodeRefs: newRoots });
+          updateDocumentInternal({ rootNodeRefs: newRoots } as RecursivePartial<Pick<ISceneDocumentInternal, 'unit'>>);
```

Comment:
you don't need to case. i updated the type of `updateDocumentInternal` and there shouldn't be type error here

Reply:
It's triggering an error in my IDE. Applied to appropriate type from what was inferred. Casting shouldn't hurt anything, but helps prevent errors. If this is the wrong type I'll gladly update it.